code/src/qik_2s12v10.cpp

```diff
@@ -37,36 +115,95 @@ namespace r2d2::moving_platform {
         hwlib::wait_ms(10);
 
         // setup the uart communication
-        usart_bus << qik_autodetect_baudrate;
+        usart_bus << qik_2s12v10_registers::autodetect_baudrate;
+    }
+
+    void qik_2s12v10_c::brake(const int8_t brake_amount) {
```

Comment:
motorspeed is not set to 0 in this function

Reply:
It is done in both the individual brake functions